### PR TITLE
Fix unit-tests for Dev15

### PR DESCRIPTION
**Pull request by AlexanderSher**

### CODE REVIEW

File: src/Package/Test/Utility/VsRHostScript.cs [hunk 1]
```diff
@@ -9,12 +10,22 @@
...
     [ExcludeFromCodeCoverage]
     public sealed class VsRHostScript : RHostScript {
         public VsRHostScript(IRSessionCallback clientApp = null) : 
-            base(VsAppShell.Current.ExportProvider.GetExportedValue<IRSessionProvider>(), clientApp) {
+            base(GetExportedValue(), clientApp) {
+        }
+
+        private static IRSessionProvider GetExportedValue() {
+            try {
+                var mock = new VsEditorAdaptersFactoryServiceMock();
```
Comment (MikhailArkhipov) [inline src/Package/Test/Utility/VsRHostScript.cs]: Is this doing something useful?

File: src/UnitTests/Core/Impl/XUnit/AssemblyMefCatalogFixture.cs [hunk 2]
```diff
@@ -17,66 +17,56 @@
...
 
                 ValidateCatalog(aggregateCatalog);
                 return aggregateCatalog;
             } finally {
                 AppDomain.CurrentDomain.AssemblyResolve -= CurrentDomain_AssemblyResolve;
+                AppDomain.CurrentDomain.AssemblyLoad -= CurrentDomain_AssemblyLoad;
             }
         }
 
+        private void CurrentDomain_AssemblyLoad(object sender, AssemblyLoadEventArgs args) {
```
Comment (MikhailArkhipov) [inline src/UnitTests/Core/Impl/XUnit/AssemblyMefCatalogFixture.cs]: Is it needed

**Comment by MikhailArkhipov** [comment]
:+1:

File: src/Package/Test/Utility/VsRHostScript.cs (hunk 1 above)
Comment (AlexanderSher) [inline src/Package/Test/Utility/VsRHostScript.cs]: Ops, that was for debugging. Will remove

**Pull request merged**